app/javascript/packages/phone-input/index.ts

```diff
@@ -186,7 +186,7 @@ export class PhoneInputElement extends HTMLElement {
       return;
     }
 
-    const phoneNumber = textInput.value as NationalNumber;
```

Comment:
I think we need to remove the dependency import above as well.

Reply:
ah yes good reminder thx

Reply:
[1ded3cd](https://github.com/18F/identity-idp/pull/10569/commits/1ded3cd05d8e7c4df573b4aed2e5246fe6d691ed)